src/test/mina_automation/executor.ml

```diff
@@ -11,83 +11,113 @@ module DockerContext = struct
     { image : string; workdir : string; volume : string; network : string }
 end
 
-type context =
-  | Dune (* application ran from dune exec command *)
-  | Local (* application ran from _build/default folder*)
-  | Debian (* application installed from mina debian package *)
-  | Docker of DockerContext.t
-  | AutoDetect
 (* application ran inside docker container *)
```

Comment:
This comment should be moved inside the Make functor. BTW it's not in the correct branch originally as well.

Reply:
Don't fully understand.. so this was previous change and it was moved to functor

Reply:
I'll comment later in this file